.github/workflows/main.yml

```diff
@@ -39,3 +39,14 @@ jobs:
 
       - name: Build packages
         run: pnpm run build
+
+      # Even though we only deploy from main, it makes sense to do this on PRs to ensure the change didn't break the Storybook
+      - name: Build storybook
+        run: pnpm run build:storybook
+
+      - name: Deploy Storybook 🚀
+        uses: JamesIves/github-pages-deploy-action@v4.3.3
+        if: github.ref == 'refs/heads/main'
```

Comment:
not needed since we actually have checks at the top of the file (that this will trigger on PR and pushes to main

Reply:
but we don't want to deploy the storybook on every PR change. Only when they're merged to main

Reply:
ooooh yeah, just read the comment wrong lol. Agreed!